packages/blade/src/components/Collapsible/CollapsibleChevronIcon.native.tsx

```diff
@@ -0,0 +1,39 @@
+import Animated, { useAnimatedStyle, useDerivedValue, withTiming } from 'react-native-reanimated';
+import { useCollapsible } from './CollapsibleContext';
+import {
+  getCollapsibleChevronIconTransforms,
+  getTransitionDuration,
+  getTransitionEasing,
+} from './commonStyles';
+import { castNativeType, makeAccessible } from '~utils';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon } from '~components/Icons';
+import { useTheme } from '~components/BladeProvider';
+
+// Not really an IconComponent, a wrapper is needed for animating the icon inside
+const CollapsibleChevronIcon: IconComponent = (props) => {
+  const { isExpanded, direction } = useCollapsible();
+  const { theme } = useTheme();
+
+  const { transformExpanded, transformCollapsed } = getCollapsibleChevronIconTransforms({
+    direction,
+  });
+  const duration = castNativeType(getTransitionDuration(theme));
+  const easing = castNativeType(getTransitionEasing(theme));
+
+  const rotateZ = useDerivedValue(() =>
```

Comment:
Why are we using useDerivedValue here? No dependencies for useDerivedValue needed here?

Reply:
Had to interpolate a string based on this value, don't need dependencies (this is being derived from `isExpanded`)